examples/1-getting-started.ipynb

```diff
@@ -0,0 +1,322 @@
+    "        return self"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So what does the transformer do? It must be able to:\n",
+    "* **initialize** itself\n",
+    "* **fit** and **transform** the incoming data prepared by the adapter; when transforming, the result should be returned as a **dictionary** that can be **passed on to the next step**\n",
+    "* **save** and **load** its parameters; this is handy when you're trying to avoid re-computing things over and over.\n",
+    "\n",
+    "See how flexible this is? You can just as easily wrap your Keras or Pytorch models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's combine our adapter and transformer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step = Step(name='classifier',\n",
+    "                       transformer=RandomForestTransformer(),\n",
+    "                       input_data=['input'],                 \n",
+    "                       adapter=input_adapter,\n",
+    "                       cache_dirpath='./cache'\n",
+    "                      )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And that's our one-step pipeline finished. You can visualize it too:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is just about the simplest pipeline you can imagine. Now let's train it!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step.clean_cache()\n",
+    "preds_train = classifier_step.fit_transform(data_train);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "acc_train = np.sum(preds_train['y_pred'] == data_train['input']['y']) / data_train['input']['y'].size"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generating test predictions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Running test data through our pipeline is as easy as this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "preds_test = classifier_step.transform(data_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "acc_test = np.sum(preds_test['y_pred'] == data_test['input']['y']) / data_test['input']['y'].size\n",
```

Comment:
@mromaniukcdl same here. I would just go with the sklearn

Reply:
+1